modules/central-services-private-config.adoc

```diff
@@ -117,6 +120,16 @@ a| The private key of the user-facing certificate that Central should use.
 | `central.adminPassword.htpasswd`
 | Administrator password for logging into {product-title}.
 This password is stored in hashed format using bcrypt.
+
+| `central.db.serviceTLS.cert`
+| An internal certificate that the Central DB service should use for deploying Central DB.
+
+| `central.db.serviceTLS.key`
+| The private key of the internal certificate that the Central DB service should use.
+
+| `central.db.password.value`
+| The password used to connect to the Central DB.
```

Comment:
Should this specify that you also need connection string for it to work?

Reply:
Will test, but for helm, I believe this will actually work

Reply:
```
helm -n stackrox install stackrox-central-services stackrox/stackrox-central-services  --set-string central.db.password.value=gorman,central.db.enabled=true 
```

```
 ks get secret central-db-password -o json | jq '.data["password"] | @base64d'
"gorman\n"
```
```
ks exec -it deploy/central-db /bin/bash
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
Defaulted container "central-db" out of: central-db, init-db (init)
[postgres@central-db-6798bbf667-dxz6v /]$ PGPASSWORD=gorman psql -d central_active
psql (13.10)
Type "help" for help.

central_active=# 
```